drivers/altcointrader.js

```diff
@@ -0,0 +1,22 @@
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+module.exports = async () => {
+  const markets = await request('https://api.altcointrader.co.za/v3/live-stats');
+
+  return Object.keys(markets).map((market) => {
+    const ticker = markets[market];
+
+    return new Ticker({
+      base: market,
+      quote: 'ZAR',
```

Comment:
Not every market has the quote `ZAR` they also have a `USDT_BTC` market listed in the end-point.